spec/rails_helper.rb

```diff
@@ -64,7 +64,7 @@
     Bullet.enable = false
   end
 
-  config.around :each, :disable_bullet do |example|
+  config.around :each do |example|
```

Comment:
No, this is the main usage, this needs to stay - otherwise disable will stop working